crates/bdk/src/types.rs

```diff
@@ -114,6 +114,11 @@ impl FeeRate {
         self.0
     }
 
+    /// Return the value as satoshi/kwu
+    pub fn as_sat_per_kwu(&self) -> f32 {
```

Comment:
using `as` usually implies the conversion was zero cost. I think here `sats_per_kwu` would be fine.

Reply:
fixed!